torch/utils/benchmark/utils/timer.py

```diff
@@ -135,21 +176,32 @@ def __init__(
         description: Optional[str] = None,
         env: Optional[str] = None,
         num_threads: int = 1,
+        language: Union[Language, str] = Language.PYTHON,
     ):
         if not isinstance(stmt, str):
             raise ValueError("Currently only a `str` stmt is supported.")
 
-        # We copy `globals` to prevent mutations from leaking, (for instance,
-        # `eval` adds the `__builtins__` key) and include `torch` if not
-        # specified as a convenience feature.
-        globals = dict(globals or {})
-        globals.setdefault("torch", torch)
-        self._globals = globals
+        # We copy `globals` to prevent mutations from leaking.
+        # (For instance, `eval` adds the `__builtins__` key)
+        self._globals = dict(globals or {})
+        if language in (Language.PYTHON, "py", "python"):
```

Comment:
nit: would be cleaner to have a function that takes care of normalizing string to the Language Enum, and then use that. I don't think it's a big deal in this patch because the logic here is still centralized, but the moment you want to have a second interface that takes in string languages, you're gonna need this refactor.